quick-start.md

```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
 
 ## Step 1: Include the TinyMCE script
 
-Include this line of code in the `<head>` of the HTML page and link to the `tinymce.min.js` source file. Substitute 'no-api-key' with your api key.
+Include the following line of code in the `<head>` of a HTML page.
 
 ```html
 <script src="{{ site.cdnurl }}" referrerpolicy="origin"></script>
 ```
 
-> Tip: A complete HTML snippet is provided in Step 2.
+## Step 2: Initialize TinyMCE as part of a web form
 
+Initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) on the web page.
 
-## Step 2: Initialize TinyMCE as part of a web form
+{{site.productname}} enables identifying replaceable elements using a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). Specify the element by passing an object that contains a `selector` to `tinymce.init()`, such as:
 
-With the script included, initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) in the web page.
+```html
+<script>
+  tinymce.init({
+    selector: '#mytextarea'
+  });
+</script>
+```
 
-Since {{site.productname}} enables identifying replaceable elements via a CSS selector, the only requirement is to pass an object that contains a `selector` to `tinymce.init()`.
+**For example**:
 
-In this example, replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance by passing the selector `'#mytextarea'` to `tinymce.init()`.
+To replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance, pass the selector `'#mytextarea'` to `tinymce.init()`.
 
 ```html
 <!DOCTYPE html>
 <html>
-<head>
-  <script src='{{ site.cdnurl }}' referrerpolicy="origin"></script>
-  <script>
-  tinymce.init({
-    selector: '#mytextarea'
-  });
-  </script>
-</head>
-
-<body>
-<h1>{{site.productname}} Quick Start Guide</h1>
-  <form method="post">
-    <textarea id="mytextarea" name="mytextarea">Hello, World!</textarea>
-  </form>
-</body>
+  <head>
+    <script src='{{ site.cdnurl }}' referrerpolicy="origin"></script>
+    <script>
+      tinymce.init({
+        selector: '#mytextarea'
+      });
+    </script>
+  </head>
+
+  <body>
+  <h1>{{site.productname}} Quick Start Guide</h1>
+    <form method="post">
+      <textarea id="mytextarea">Hello, World!</textarea>
+    </form>
+  </body>
 </html>
 ```
 
+Adding this content to a HTML file and opening it in a web browser will load a TinyMCE editor, such as:
+
+{% include codepen.html id="default" %}
+
+To remove the notice:
+
+<img alt="warning icon" src="{{ site.baseurl }}/images/icons/warning.svg" style="vertical-align:middle;"/>**_This domain is not registered with {{site.cloudname}}..._**
+
+Replace `no-api-key` in the source script (`<script src=...`) with a [{{site.cloudname}} API key]({{site.accountsignup}}).
```

Comment:
Should this say "...which you can get by signing up..." or something? The next sentence implies it but wondering if we should be more... straightforward?

Reply:
fixed?